make_benchmark_data/process_benchmark_scrape.R

```diff
@@ -0,0 +1,71 @@
+library(jsonlite)
+library(data.table)
+
+
+read_json <- function(infile){
+
+    json_res = fromJSON(infile)
+    json_res = data.frame(json_res)
+    return(json_res)
+
+}
+
+process_body <- function(in_df){
+
+    library(stringr)
+    # remove redundant white space
+    in_df$body = str_squish(in_df$body)
+
+    # convert unicode stuff
+    ## TODO: Do this for all languages, JIC
+    in_df$body = gsub("\u2019", "'", in_df$body)
```

Comment:
```suggestion
    in_df$body = gsub("(\u2019|\u2018|\u201c|\u201d|\u2014)", "'", in_df$body)
```
gsub supports [regular expressions](https://www.rexegg.com/regex-quickstart.html) so this may save you having to write multiple lines. You can add more to above, but wanted to suggest an example.